extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_index.php

```diff
@@ -23,20 +23,23 @@
 						'wds-icons-magnifying-glass',
 						'wds-icon wds-icon-small'
 					) ?>
-					<input class="wds-global-navigation__search-input" name="search" placeholder="<?= DesignSystemHelper::renderText( $model['search']['module']['placeholder-inactive'] ); ?>"/>
+					<input class="wds-global-navigation__search-input"
+						name="<?= Sanitizer::encodeAttribute( $model['search']['module']['results']['param-name'] ); ?>"
+						data-active-text="<?= DesignSystemHelper::renderText( $model['search']['module']['placeholder-active'] ); ?>"
+						placeholder="<?= DesignSystemHelper::renderText( $model['search']['module']['placeholder-inactive'] ); ?>"/>
 				</label>
-				<button class="wds-button wds-is-text wds-global-navigation__search-close">
+				<button class="wds-button wds-is-text wds-global-navigation__search-close" type="button">
```

Comment:
It's one of the three valid types for a button (with submit and reset) and by adding it, it stops it defaulting to also being a submit button. I originally used reset, but I'm not sure if we want this to also clear the entered search text of the user, or just collapse the search box. If we also want the user's text to be cleared, this can be `type="reset"`.